.circleci/config.yml

```diff
@@ -514,12 +513,9 @@ jobs:
             - when:
                 condition:
                   not:
-                    equal: ["", << parameters.codecov_args >>]
+                    equal: ["", << parameters.codecov_flags >>]
                 steps:
-                  - run:
-                      name: Upload coverage to codecov
-                      command: |
-                        nox -s codecov -- << parameters.codecov_args >>
+                  - codecov/upload: {flags: <<parameters.codecov_flags>>}
```

Comment:
I saw that the codecov orb has a `when` argument so in theory we could use it, up to you ofc